test/reline/helper.rb

```diff
@@ -19,12 +19,12 @@
 end
 
 module Reline
-  class <<self
+  class << self
     def test_mode(ansi: false)
       @original_iogate = IOGate
 
-      if ENV['RELINE_TEST_ENCODING']
-        encoding = Encoding.find(ENV['RELINE_TEST_ENCODING'])
+      if Reline.const_defined?(:RELINE_TEST_ENCODING) && RELINE_TEST_ENCODING.is_a?(Encoding)
```

Comment:
RELINE_TEST_ENCODING is a top level constant. If it is defined, the value is always an Encoding.
I think one of these is better
`if Object.const_defined?(:RELINE_TEST_ENCODING)`
`if defined?(RELINE_TEST_ENCODING)`